flintrock/config.yaml.template

```diff
@@ -1,6 +1,8 @@
 services:
   spark:
     version: 1.6.0
+    # distribution: # optional; default to '2.6'
+    # download-source: # optional; default to 'https://s3.amazonaws.com/spark-related-packages/spark-${version}-bin-hadoop${distribution}.tgz'
```

Comment:
I prefer the variable substitution to be done in Python and not Bash, so the template variable should be `{version}` and not `${version}`. This is consistent with how Flintrock substitutes variables in the config templates, for example.

Same style nitpicks as above.